packages/wonder-blocks-modal/components/one-pane-dialog/one-pane-dialog.md

```diff
@@ -0,0 +1,125 @@
+        width: 874,
+        height: 551,
+        position: "absolute",
+        top: -260,
+        left: -420
+    },
+});
+
+const StyledContainer = addStyle("div");
+
+<View style={styles.previewSizer}>
+    <View style={styles.modalPositioner}>
+        <OnePaneDialog
+            title="Title"
+            content={
+                <View style={styles.modalContent}>
+                    <Body>
+                        {"Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est."}
+                    </Body>
+                </View>
+            }
+            footer={<View>
+                <Button type="button">Button (no-op)</Button>
+            </View>}
+            onClose={() => alert("This would close the modal.")}
+            above={<StyledContainer style={styles.aboveStyles} />}
+        />
+    </View>
+</View>;
+```
+
+### Example: Long content (overflows)
+
+```js
+const {StyleSheet, css} = require("aphrodite");
+const {View} = require("@khanacademy/wonder-blocks-core");
+const {Title, Body} = require("@khanacademy/wonder-blocks-typography");
+const Button = require("@khanacademy/wonder-blocks-button").default;
+
+const styles = StyleSheet.create({
+    previewSizer: {
+        height: 512,
+    },
+
+    modalPositioner: {
+        // Checkerboard background
+        backgroundImage: "linear-gradient(45deg, #ccc 25%, transparent 25%), linear-gradient(-45deg, #ccc 25%, transparent 25%), linear-gradient(45deg, transparent 75%, #ccc 75%), linear-gradient(-45deg, transparent 75%, #ccc 75%)",
+        backgroundSize: "20px 20px",
+        backgroundPosition: "0 0, 0 10px, 10px -10px, -10px 0px",
+
+        flexDirection: "row",
+        alignItems: "center",
+        justifyContent: "center",
+
+        position: "absolute",
+        left: 0,
+        right: 0,
+        top: 0,
+        bottom: 0,
+    },
+});
+
+<View style={styles.previewSizer}>
+    <View style={styles.modalPositioner}>
+        <OnePaneDialog
+            title="Title"
+            subtitle="Wow, look at all this content!"
+            content={
+                <View>
+                    <Body>
+                        {"Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est."}
+                    </Body>
```

Comment:
You may want to give https://github.com/knicklabs/lorem-ipsum.js for these paragraphs.

Reply:
I will try it as part of PR 3/3 :)